com.unity.render-pipelines.core/ShaderLibrary/Color.hlsl

```diff
@@ -408,7 +420,11 @@ real3 LinearToLogC(real3 x)
 #endif
 }
 
+#if defined(SHADER_API_SWITCH) // Must use full float precision on Switch to avoid artefact when using ACES tonemapping
+float3 LogCToLinear(float3 x)
+#else
 real LogCToLinear_Precise(real x)
```

Comment:
This looks wrong, LogCToLinear_Precise will not be defined for SWITCH.

Reply:
Thanks, I should have checked this.
I made the fixed and confirmed this is loading on Switch.